core/src/test/java/com/scalar/db/transaction/consensuscommit/CommitMutationComposerTest.java

```diff
@@ -220,6 +239,52 @@ public void add_PutAndNullResultGiven_ShouldComposePutWithPutIfCondition()
     assertThat(actual).isEqualTo(expected);
   }
 
+  @Test
+  public void add_PutWithoutCkAndPreparedResultGiven_ShouldComposePutWithPutIfCondition()
+      throws ExecutionException {
+    // Arrange
+    Put put = preparePutWithoutCk();
```

Comment:
PUT operation doesn't have the issue. So this test isn't for reproducing something, but just for consistency with DELETE.